### DISCUSSION TITLE
Can not switch from 7.4 to 8.0 or 8.1 after trying to add 8.1

### DISCUSSION THREAD

**Discussion by lambertbeekhuis** [shivammathur/homebrew-php#781] [Q&A / Help]
I am having Big Sur, and I used to be able to switch between php 7.4 and 8.0. Very nice!!!

But... I am stuck on 7.4 only after trying to add 8.1. 

Here is what i Idid:

>> brew update

>> brew install shivammathur/php/php@8.1

>> brew link --overwrite --force shivammathur/php/php@8.1

I tried to switch to 7.4 or 8.0, but for I am stuck on PHP 7.4 now, and cannot switch either to 8.0 or 8.1
(yes, I closed and opened the terminal)

During the process somewhere I was asked to update 7.4 to my $PATH, so I did. I was also asked brew unlink x and link, so I did.

I am confused where I am now, and how to get it running again

Currently my $PATH (echo $PATH)is:
/usr/local/opt/php@7.4/sbin:/usr/local/opt/php@7.4/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbinx:/Library/Apple/usr/bin:/sbin

If I do 'brew search php' I find many PHP's:

brew-php-switcher                        phpstan
php                                      phpunit
php-code-sniffer                         shivammathur/php/php
php-cs-fixer                             shivammathur/php/php@5.6 ✔
php-cs-fixer@2                           shivammathur/php/php@7.0
php@7.2                                  shivammathur/php/php@7.1
php@7.3                                  shivammathur/php/php@7.2
php@7.4 ✔                                shivammathur/php/php@7.3
php@8.0 ✔                                shivammathur/php/php@7.4 ✔
phpbrew                                  shivammathur/php/php@8.0 ✔
phplint                                  shivammathur/php/php@8.2
phpmd                                    pcp
phpmyadmin                               pup

(I accidently did

>> brew reinstall $(brew deps shivammathur/php/php@5.6) shivammathur/php/php@8.0

mixing 5.6 and 8.0 in one command, so maybe this is the reason 5.6 is checked)

I use apache by the way.

How can I add 8.0 and 8.1 to my Big Sur again?

**Comment by shivammathur** [comment]
Please remove `/usr/local/opt/php@7.4/sbin` and `/usr/local/opt/php@7.4/bin` from the `PATH`. and then run 
`brew link --overwrite --force shivammathur/php/php@8.1` to switch to `PHP 8.1`.

When you run `brew link` for a PHP version it is linked in brew's `bin` and `sbin` directories i.e. `/usr/local/bin` and `/usr/local/sbin` in your case, so it is not required add specific PHP version binaries to `PATH`.
